src/stories/StyledDataTextRow.tsx

```diff
@@ -0,0 +1,16 @@
+import { PropsWithChildren } from 'react';
+
+interface IStyledDataTextRowProps extends PropsWithChildren {
+  label: string;
+}
+
+export default function StyledDataTextRow({ label, children }: IStyledDataTextRowProps) {
+  return (
+    <div className="flex">
```

Comment:
Does this work with `className="flex"`? Normally we use the class attribute to add CSS classes (className is used in Javascript).

Reply:
well, since this is JSX and not HTML, all attributes with dashes will be camelCase and class should be named into className

Reply:
Ah, makes sense 👍  thanks 😉